File: Python/ex9.py
# 9. Check if a word or a number is a palindrome

def palindrome(list):
    n = len(list)
    for i in range(0,int(n/2)):
        if list[i] != list[n-i-1]:
            return False
    return True
a = [1,2,2,1]
b = palindrome(a)
if (b):
    print("yes")
else:
    print("no")

File: JavaScript/ex9.js
// 9. Check if a word or a number is a palindrome

const prompt = require ("prompt-sync")();

function palindrome(list){
    for (i = 0; i < list.length/2; i++){
        if (list[i] != list[list.length - 1 - i]){
            return false;
        }
    }
    return true;
}

out =  palindrome(["a", "b", "b", "a"])
if (out){
    console.log("it is a palindrome")
}
else {
    console.log("it is not a palindrome")
}